livekit/src/room/mod.rs

```diff
@@ -716,8 +722,13 @@ impl RoomSession {
         }
 
         let (participant_sid, stream_id) = lk_stream_id.unwrap();
+        let mut track_id = track.id();
+        if stream_id.starts_with("TR") {
+            track_id = stream_id.into();
+        }
```

Comment:
I guess this logic on the JS side was just for Firefox? Which seems unneeded here?

Reply:
should it always be stream_id or track_id, then?

Reply:
not just for firefox.. this is required as by protocol v10: https://github.com/livekit/client-sdk-js/pull/881

Reply:
>=v10, we have two ways of packing the trackID, and clients need to support both